packages/plugin/src/ChartPlugin.js

```diff
@@ -95,7 +95,7 @@ class ChartPlugin extends Component {
 
         try {
             const visualization =
-                Object.keys(config).length === 1
+                forDashboard || Object.keys(config).length === 1
```

Comment:
could you add a unit test for when forDashboard is true?

Reply:
Added.